sdk/python/kfp/compiler/compiler_test.py

```diff
@@ -804,6 +805,30 @@ def hello_world(text: str) -> str:
             pipeline_spec['root']['inputDefinitions']['parameters']['text']
             ['defaultValue'], 'override_string')
 
+    def test_compile_container_component_simple(self):
+
+        @dsl.container_component
+        def hello_world_container() -> str:
+            """Hello world component."""
+            return ContainerSpec(
+                image='python3.7',
+                command=['echo', 'hello world'],
+                args=[],
+            )
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            output_json = os.path.join(tempdir, 'component.yaml')
+            compiler.Compiler().compile(
+                pipeline_func=hello_world_container,
+                package_path=output_json,
+                pipeline_name='hello-world-container')
+            with open(output_json, 'r') as f:
+                pipeline_spec = yaml.safe_load(f)
+        self.assertEqual(
+            pipeline_spec['components']['comp-hello-world-container']
+            ['outputDefinitions']['parameters']['Output']['parameterType'],
+            "STRING")
```

Comment:
nit: quote consistency

```suggestion
            'STRING')
```